src/GoogleMaps/Marker.elm

```diff
@@ -1,6 +1,6 @@
 module GoogleMaps.Marker exposing
-    ( Marker, Latitude, Longitude, init
-    , withIcon, withDraggableMode, withTitle
+    ( Marker, Latitude, Longitude, init, Animation(..)
```

Comment:
```suggestion
    ( Marker, Latitude, Longitude, init, Animation
    , animationDrop, animationBounce, animationNone
```

By not exposing the union's entries, we can add new ones later without breaking the API.

Reply:
I want to clarify this change.  Adding to the union would break the API because anyone making a case statement with it would then have a broken case but are there other reasons not to expose the contents of a union type?  Is that the only breakage that could occur?  Not that that isn't bad enough.  That would suck for 3rd parties using the library.

Is it fairly standard practice to not actually expose the contents of a union type when developing a library?  I've not thought about this from the perspective of writing code that people outside my team will use.

Reply:
I'm going to make this change locally because it implies some new functions I should add.

Reply:
it's fairly standard to not expose unions unless you really want your users to be able to pattern match

check elm's http library, there's two unexposed unions there, Body and Expect